src/app/components/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+"use client"
+import { Header, PrimaryNav } from "@trussworks/react-uswds";
+import Link from "next/link";
+import Image from "next/image";
+import { usePathname } from "next/navigation";
+import React from "react";
+import styles from "../styles/Home.module.scss"
+
+interface NavigationLinkProps {
+  href: string;
+  text: string;
+  onClick?: () => void;
+}
+
+export default function Footer({ }) {
+    const basePath = "/dibbs-site";
+
+    const [expanded, setExpanded] = React.useState(false)
+    const onClick = () => { if (window.innerWidth < 1024) setExpanded((prvExpanded) => !prvExpanded) }
+
+    const testItemsMenu = [
+        <Link
+            href='/'
+            key="one"
+            className={`usa-nav__link ${styles.homeNavItem}`}
+            onClick={onClick}>
+
+            <span className={styles.navbarItemText}>Home</span>
+
+        </Link>,
+        <NavigationLink key="two" href={`${basePath}/`} text='Home' onClick={onClick} />,
+        <NavigationLink key="two" href='/our-products' text='Our products' onClick={onClick} />,
+        <NavigationLink key="two" href='/our-products' text='Case studies' onClick={onClick} />,
```

Comment:
Here's an example of the minor change I mentioned if we modify the `NavigationLinkProps` type.

```suggestion
    <NavigationLink key="two" href='/our-products' onClick={onClick}>
      Case studies
    </NavigationLink>,
```